### PR TITLE
ci: start running some of the citgm projects [v2]

### PR DESCRIPTION
**Pull request by nektro**
supersedes https://github.com/oven-sh/bun/pull/14351

### CODE REVIEW

**Comment by robobun** [comment]
@nektro, your commit b040b454ac02645f303bb378f8706233b7e180c0 is building: [`#4597`](https://buildkite.com/bun/bun/builds/4597)

File: src/cli/pm_trusted_command.zig [hunk 1]
```diff
@@ -169,16 +169,17 @@ pub const TrustCommand = struct {
...
-            Output.errGeneric("0 scripts ran. This means all dependencies are already trusted or none have scripts.", .{});
+            Output.warn("0 scripts ran. This means all dependencies are already trusted or none have scripts.", .{});
         } else {
-            Output.errGeneric("0 scripts ran. The following packages are already trusted, don't have scripts to run, or don't exist:\n\n", .{});
+            Output.warn("0 scripts ran. The following packages are already trusted, don't have scripts to run, or don't exist:\n\n", .{});
             for (packages_to_trust) |arg| {
                 Output.prettyError(" <d>-<r> {s}\n", .{arg});
             }
         }
+        Global.exit(0);
```
Comment (Jarred-Sumner) [inline src/cli/pm_trusted_command.zig]: Whats the rationale? This is a breaking change

**Comment by Jarred-Sumner** [comment]
Where are these run and how can the results be viewed? I don't see it in the log

File: src/cli/pm_trusted_command.zig (hunk 1 above)
Comment (nektro) [reply src/cli/pm_trusted_command.zig]: its exactly the same behavior as before, i just moved this call from the callers into this function

**Comment by nektro** [comment]
image

File: src/cli/pm_trusted_command.zig (hunk 1 above)
Comment (nektro) [reply src/cli/pm_trusted_command.zig]: oh i see; it not being an error. it didnt seem like something that should be an error condition and i consulted with dylan

Comment (nektro) [reply src/cli/pm_trusted_command.zig]: the runner script runs this command and having this be an error makes it indistinguishable from having scripts that failed/crashed vs there were none in the first place

Comment (nektro) [reply src/cli/pm_trusted_command.zig]: https://github.com/search?q=%22bun+pm+trust+--all%22&type=code looks like there's not a ton of in-the-wild usage and there's even multiple counts of users doing `|| true` potentially to obtain this behavior on their own

**Comment by nektro** [comment]
superseded

**Pull request closed without merging**